src/core/timsort.ts

```diff
@@ -217,23 +215,12 @@ function gallopRight<T>(value: T, array: T[], start: number, length: number, hin
 
 function TimSort<T>(array: T[], compare: CompareFunc<T>) {
     let minGallop = DEFAULT_MIN_GALLOPING;
-    let length = 0;
-    let tmpStorageLength = DEFAULT_TMP_STORAGE_LENGTH;
-    let stackLength = 0;
     let runStart: number[];
     let runLength: number[];
     let stackSize = 0;
 
-    length = array.length;
-
-    if (length < 2 * DEFAULT_TMP_STORAGE_LENGTH) {
-        tmpStorageLength = length >>> 1;
-    }
-
     var tmp: T[] = [];
 
-    stackLength = length < 120 ? 5 : length < 1542 ? 10 : length < 119151 ? 19 : 40;
```

Comment:
Unused parameter. I checked the [original code](https://github.com/mziccard/node-timsort/blob/master/src/timsort.js#L438) that `stackLength` was used to init array but we don't use it so it can be safely removed.